### PR TITLE
Style refactor

### PR DESCRIPTION
**Pull request by SUPERCILEX**
@samtstern I started working on making my anonymous auth PR support phone auth, but I kept shuddering some really gross phone auth code so this PR fixes that messiness. 😄

The PR
- Auto styles all documentation so it's nice and aligned and stuff
- Restyles the manifests with the trailing space
- Cleans up phone auth code
- Deletes a whole bunch of zombie code we weren't using (Yay! 😄)

### CODE REVIEW

File: auth/src/main/java/com/firebase/ui/auth/ui/TermsTextView.java [hunk 1]
```diff
@@ -1,50 +0,0 @@
...
-import android.util.AttributeSet;
-
-import com.firebase.ui.auth.ui.email.PreambleHandler;
-
-/**
- * Text view to display terms of service before completing signup.
- * The view helps display TOS linking to the provided custom URI.
- * It handles the styling of the link and opens the uri in a CustomTabs on click.
- */
-public class TermsTextView extends AppCompatTextView {
```
Comment (SUPERCILEX) [inline auth/src/main/java/com/firebase/ui/auth/ui/TermsTextView.java]: This is the only non-style change, I got rid of the custom TextView.

File: auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneVerificationActivity.java [hunk 2]
```diff
@@ -307,52 +309,48 @@ private void signingWithCreds(@NonNull PhoneAuthCredential phoneAuthCredential)
...
+                                    finish(authResult.getUser());
+                                }
+                            }
+                        }, SHORT_DELAY_MILLIS);
                     }
-                }, SHORT_DELAY_MILLIS);
-            }
-        }).addOnFailureListener(this, new OnFailureListener() {
+                }).addOnFailureListener(this, new OnFailureListener() {
             @Override
```
Comment (samtstern) [inline auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneVerificationActivity.java]: nit: this indentation is strange, the onFailure method is to the left of addOnFailureListener

File: auth/src/main/java/com/firebase/ui/auth/ui/phone/SubmitConfirmationCodeFragment.java [hunk 3]
```diff
@@ -134,7 +134,7 @@ public void onSaveInstanceState(Bundle outState) {
 
     private void setTimer(long millisUntilFinished) {
         mCountDownTextView.setText(String.format(getString(R.string.resend_code_in),
-                timeRoundedToSeconds(millisUntilFinished)));
+                                                 timeRoundedToSeconds(millisUntilFinished)));
```
Comment (samtstern) [inline auth/src/main/java/com/firebase/ui/auth/ui/phone/SubmitConfirmationCodeFragment.java]: If this was auto-formatted, can we change the rules not to do these aligned hanging indents (here and everywhere)?  They don't survive refactoring.

**Comment by SUPERCILEX** [comment]
@samtstern Done with a green build! 😄

**Pull request merged**